ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -283,6 +291,7 @@ export default class TaskRunsCard extends Component {
     const { showArtifacts, showMore } = this.state;
     const run = this.getCurrentRun();
     const liveLogArtifact = this.getLiveLogArtifactFromRun(run);
+    const hashInUrl = window.location.hash === '#artifacts';
```

Comment:
`const showArtifacts = window.location.hash === '#artifacts';` then you can remove `showArtifacts` from `state`.

Reply:
Sure👍